lightstep/resource_opsgenie_destination.go

```diff
@@ -0,0 +1,128 @@
+package lightstep
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/lightstep/terraform-provider-lightstep/client"
+)
+
+func resourceOpsgenieDestination() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceOpsgenieDestinationCreate,
```

Comment:
do you want to add a note for future readers about why there's no Update entry here?